app/javascript/components/search/results/StudySearchResult.js

```diff
@@ -15,16 +15,33 @@ export function formatDescription(rawDescription, term) {
   return shortenDescription(textDescription, term)
 }
 
+/** Highlight matched words, keeping original capitalization of text */
+function highlightWords(text, termMatches) {
+  let stylizedText = ''
+  const words = text.split(' ')
+  words.forEach(word => {
+    let alreadyMatched = false
+    termMatches.forEach(term => {
+      if (term.toUpperCase() === word.toUpperCase()) {
+        stylizedText = `${stylizedText} <span class='highlight'>${word}</span>`
+        alreadyMatched = true
+      }
+    })
+    if (!alreadyMatched) {
+      stylizedText = `${stylizedText} ${word}`
+    }
```

Comment:
I think you could make this a little cleaner by doing the code below.  It also has the advantage of being a bit more robust to duplicate term matches (IDK if that's actually a concern though)
```suggestion
  let stylizedWord = word
  termMatches.forEach(term => {
    if (term.toUpperCase() === word.toUpperCase()) {
      stylizedWord = `<span class='highlight'>${word}</span>`
    }
  })
  stylizedText = `${stylizedText} ${stylizedWord}`
```